src/192/main/kotlin/org/rust/ide/hints/RsPlainTypeHint.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.hints
+
+import com.intellij.codeInsight.hints.InlayInfo
+import com.intellij.codeInsight.hints.Option
+import com.intellij.psi.PsiElement
+
+/** New interactive type hints are used instead */
```

Comment:
Maybe add an (registry) option so that the user can switch between old and new type hints?

Reply:
Let's make sure that new type hints cover all the functionality of the old ones instead.